src/domain/consents/revoke.ts

```diff
@@ -57,6 +65,9 @@ export async function revokeConsentStatus (
   return consent
```

Comment:
you shouldn't update Consent if it is already REVOKED and where is an update in DB invocation? you should add guard if(consent.status !== 'ACTIVE') { throw ... }

Reply:
so at the  moment we're not updating consent if it's already revoked. we just skip the updating/revoking and send the PATCH request directly - since there's a `revokedAt` field the assumption is the PISP/DFSP can see that
Lmk if that seems right